park-dashboard/src/Pages/Generic Charts/GenericCharts.js

```diff
@@ -35,6 +35,14 @@ const GenericChart = () => {
           </Col>
         </Row>
       </Container>
+      <Container style={{ marginBottom: "5rem", marginTop: "12rem"}}>
+        <Col>
+          <h2 style={{ fontSize: "18px" }} className="text-center">
+            3D Pie Chart: Alerts Vs Park Counts
+          </h2>
+          <AlertsVsParks />
```

Comment:
I noticed a crash in this component when I was tapping on `Park Closure`. Can you fix this?

Steps to reproduce
- Go to Generic charts
- Scroll down to `AlertsVsParks` component
- Tap on `Park Closure`

<img width="1508" alt="image" src="https://github.com/rutujapadgilwar/Park-Dashboard/assets/114516950/b557798e-68d1-437b-ba10-b7df34fb5bf8">

Reply:
Hi @juhiaugustine, thank you for bringing this to my attention. I have addressed the issue and added null value validation to ensure proper functionality.